oar/cli/cmd_image_signed_check.py

```diff
@@ -3,44 +3,82 @@
 import re
 import requests
 from requests.exceptions import RequestException
-from bs4 import BeautifulSoup
 from oar.core.const import *
 from oar.core.worksheet_mgr import WorksheetManager
-
+import time
 
 logger = logging.getLogger(__name__)
 
+
+# 1. go to main URL and grab all links that matches the build type
+# 2. construct a complete URL,
+def get_image_digest(url, current_try=0):
+    logger.info(f"Getting digest from {url}")
+    res = requests.get(url)
+    if not res.ok:
+        logger.info(f"CODE: {res.status_code} -- {res.reason}")
+        res.raise_for_status
+    change_log_json = res.json().get("changeLogJson")
+    if not change_log_json:
+        logger.info("No changeLogJson element is found!")
+        raise requests.InvalidJSONError
```

Comment:
suggest to add import for `InvalidJSONError`. and do `raise InvalidJSONError("No changeLogJson element is found!")` here

`from requests.exceptions import InvalidJSONError`

Reply:
done.